src/base/system_parameter.c

```diff
@@ -5799,6 +5807,18 @@ static SYSPRM_PARAM prm_Def[] = {
    (char *) NULL,
    (DUP_PRM_FUNC) NULL,
    (DUP_PRM_FUNC) NULL},
+  {PRM_ID_REPL_SEMISYNC_ACK_MODE,
+   PRM_NAME_REPL_SEMISYNC_ACK_MODE,
+   (PRM_FOR_SERVER),
+   PRM_KEYWORD,
```

Comment:
More is needed for mature keyword parameters implementation. Please see functions:

- prm_print
- sysprm_print_sysprm_value
- sysprm_generate_new_value